tests/tests/kernel/datatypes/ezuser/ezusertype_regression.php

```diff
@@ -188,11 +195,13 @@ public function testUpdatePasswordUpdatesSerializedData()
     {
         $userId = $this->userObject->attribute('id');
 
-        $passwordHash = $this->getSerializedPasswordHash($this->userObject);
+        $user = ezuser::fetch( $userId );
+        $passwordHash = $user->PasswordHash;
```

Comment:
Could you explain this change, please?

Reply:
getSerializedPasswordHash was checking the 'data_text' value of the datatype. But the function eZUserOperationCollection::password is just updating the ezuser DB table. I changed the the test to more directly check the eZUserOperationCollection::password function.

The 'data_text' value in the datatype is only a temporarily place to store user data for draft versions. If an admin creates a new users, you don't want to have the draft data in the ezuser DB table right away.